lnwallet/channel.go

```diff
@@ -7667,6 +7667,18 @@ type LocalForceCloseSummary struct {
 	// commitment state.
 	CloseTx *wire.MsgTx
 
+	// ChanSnapshot is a snapshot of the final state of the channel at the
+	// time the summary was created.
+	ChanSnapshot channeldb.ChannelSnapshot
+
+	// Resolutions contains all the data required for resolving the
+	// different output types of a commitment transaction.
+	Resolutions fn.Option[Resolutions]
```

Comment:
I went for this approach here, it would be to much of a change to introduce options for all the separate types like for example:

AnchorResolution fn.Option[AnchorResolution] ...

because we use the `nil` case a lot and also pass it into other structures. Maybe a deeper refactor in the log run but not now.